templates/git-sync-relay/git-sync-relay-ingress.yaml

```diff
@@ -0,0 +1,62 @@
+##########################
+# git-sync-relay ingress #
+##########################
+{{- if and
+  .Values.gitSyncRelay.enabled
+  (eq .Values.gitSyncRelay.repoFetchMode "webhook")
+}}
```

Comment:
```suggestion
{{- if and .Values.gitSyncRelay.enabled (eq .Values.gitSyncRelay.repoFetchMode "webhook") }}
```

Reply:
IMHO the multi-line version is so much easier to grok at a glance.